config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  resources :users, only: [:new, :create]
+  resources :users, only: %i[new create show]
```

Comment:
I would disable this in Rubocop and go back to the normal array syntax

Reply:
This is fixed in sha: edafaec